Altis_Life.Altis/core/functions/TON/fn_sendMessage.sqf

```diff
@@ -0,0 +1,70 @@
+/*
+    File: fn_sendMessage.sqf
+    Author: blackfisch
+
+    Description: send message to target
+*/
+params [
+    ["_ehParams", [], []],
```

Comment:
Shouldn't that be `["_ehParams", [], [[]]],`

Reply:
whoops, yep :D consider that one fixed ^^